sled-agent/src/services.rs

```diff
@@ -79,7 +88,11 @@ impl ServiceManager {
             log,
             config_path,
             zones: Mutex::new(vec![]),
-            vnic_allocator: VnicAllocator::new("Service", physical_link)?,
+            vnic_allocator: VnicAllocator::new(
+                "Service",
+                physical_link.clone(),
```

Comment:
Does `physical_link` imply that we've not yet gotten to the point of hanging zone VNICs off of etherstub devices yet?

Reply:
That’s right. The sled agent currently creates all VNICs over a physical link that can be optionally specified in a configuration file. It defaults to the first physical link on the system. I still need to make the issue tracking the work to put all those VNICs over an ether stub.

Reply:
Thanks @smklein!